cores/esp8266/core_esp8266_sigma_delta.c

```diff
@@ -0,0 +1,152 @@
+#include "Arduino.h" // using pinMode
+
+// definitions in esp8266_peri.h style
+#define GPSD  ESP8266_REG(0x368) // GPIO_SIGMA_DELTA register @ 0x600000368
+#define GPSDT 0  // target, 8 bits
+#define GPSDP 8  // prescaler, 8 bits
+#define GPSDE 16 // enable
+
+/******************************************************************************
+ * FunctionName : sigmaDeltaEnable
+ * Description  : enable the internal sigma delta source
+ * Parameters   : none
+ * Returns      : none
+*******************************************************************************/
+void ICACHE_FLASH_ATTR sigmaDeltaEnable()
+{
+  GPSD = (0 << GPSDT) | (0 << GPSDP) | (1 << GPSDE); //SIGMA_DELTA_TARGET(0) | SIGMA_DELTA_PRESCALER(0) | SIGMA_DELTA_ENABLE(ENABLED)
+}
+
+/******************************************************************************
+ * FunctionName : sigmaDeltaDisable
+ * Description  : stop the internal sigma delta source
+ * Parameters   : none
+ * Returns      : none
+*******************************************************************************/
+void ICACHE_FLASH_ATTR sigmaDeltaDisable()
+{
+  GPSD = (0 << GPSDT) | (0 << GPSDP) | (0 << GPSDE); //SIGMA_DELTA_TARGET(0) | SIGMA_DELTA_PRESCALER(0) | SIGMA_DELTA_ENABLE(DISABLED)
+}
+
+/******************************************************************************
+ * FunctionName : sigma_delta_attachPin
```

Comment:
Some FunctionName comments got updated to camelCase, but some like this did not.